plugins/catalog-backend/migrations/20210302150147_refresh_state.js

```diff
@@ -20,39 +20,34 @@
  * @param {import('knex').Knex} knex
  */
 exports.up = async function up(knex) {
+  let STRING_TEXT = 'text';
+  if (knex.client.config.client.includes('mysql')) {
+    STRING_TEXT = 'string';
+  }
```

Comment:
Should get rid of this - not really needed right? `@ts-ignore` should be avoided like the plague anyway :)

Reply:
Postgres 9, will fail in tests if we use string, and MySQL will fail if we use text. For postgres 9, for whatever reason, the order of the entityRefs will be different if it's "string", and MySQL cannot index on "text". The safest solution seems to be to use "string" for MySQL and "text" for Postgres. I have not found a workaround to use "string" that satisfies all DB's.

Reply:
Ah, yeah, much cleaner too. Thx!